packages/terra-image/src/terra-dev-site/test/image/ImageFluid.test.jsx

```diff
@@ -15,11 +15,11 @@ const ImageFluid = () => (
     </div>
     <p>Image width is larger than the container. Image height is smaller than the container.</p>
     <div id="height-smaller" className={cx('image-conatiner-reduced-width')}>
-      <Image src={placeholder150x150} alt="rounded" variant="rounded" isFluid />
+      <Image src={placeholder150x150} alt="rounded" variant="rounded" imageFit="cover" isFluid />
```

Comment:
@pranav300 prop typo:  imageFit --> fit="cover" (line 18 and line 22) so not actually applying in this test

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/18dd5b93d46d856d085bf005ebcac7b4ac2b3685